src/clj_rn/shell.clj

```diff
@@ -10,7 +10,7 @@
 
 (defn exec
   [command]
-  (let [process (.exec (Runtime/getRuntime) (into-array (str/split command #" ")))]
+  (let [process (.exec (Runtime/getRuntime) (into-array ["sh" "-c" command]))]
```

Comment:
Can you detail this? How will windows be impacted here?

Reply:
`Runtime/getRuntime` require command as an array.
This command
```
command run --first-arg="something here" --second-arg="something here" --last-arg
```

should be separated like this
```
command
run 
--first-arg="something here"
--second-arg="something here" 
--last-arg
```
 I can split string with this regexp `/\S+(?=(?:[^\"]*\"[^\"]*\")*[^\"]*$)/' ` it it should work on windows.